<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogStokBarang</class>
 <widget class="QDialog" name="DialogStokBarang">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>#</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Kode barang&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtKode"/>
   </item>
   <item>
    <widget class="QLabel" name="lblJumlah">
     <property name="text">
      <string>##JUMLAH YANG AKAN DITAMBAH/DIAMBIL (satuan)##</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtJumlah"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>12</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="btnLanjut">
     <property name="text">
      <string>##TAMBAH/AMBIL STOK##</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
